libpod/runtime_ctr.go

```diff
@@ -329,6 +331,13 @@ func (r *Runtime) removeContainer(ctx context.Context, c *Container, force bool)
 		return errors.Wrapf(ErrCtrExists, "container %s has dependent containers which must be removed before it: %s", c.ID(), depsStr)
 	}
 
+	var volumes []string
+	if removeVolume {
+		volumes, err = c.namedVolumes()
+		if err != nil {
+			logrus.Errorf("unable to retreive builtin volumes: %v", err)
```

Comment:
s/retreive/retrieve

Reply:
It would probably be good to add the container id to the message:
`logrus.Errorf("unable to retrieve builtin volumes %v for container %v", err, c.ID())

Reply:
Fixed